tsightler/ring-mqtt
# Update to 1.0.0

Update to 1.0.0 release
* Update to work with latest ring alarm API changes
* Fix bugs around republish
* Add timed republish during startup (should help with auto discovery issues during upgrades and on HASS.io startup
* Lots of minor code refactoring to simplify and move code to functions from main
* Change topic level order to prepare for possibility of supporting non-alarm devices (cameras/lighting)
